src/plugins/CoreLoadPlugin.ts

```diff
@@ -1,10 +1,17 @@
 import * as path from 'path';
+import { getBasePath, resolveMid } from './util/main';
+import { CallExpression, Program } from 'estree';
+import { getNextItem } from './util/parser';
+import Set from '@dojo/shim/Set';
```

Comment:
I had to do the same thing with `Map` in `I18nPlugin` due to type collisions in what was either an update to TS or to `@dojo/shim` (I can't remember exactly which).